### PR TITLE
SSVD-2858: Configure ZoneStatus reporting for Device Health and Hub Offline usecases 

### PR DESCRIPTION
**Pull request by tpmanley**
Previously we were configuring the Temperature attribute for reporting at
5 minute intervals so that we could relatively quickly determine if the
device is offline for Device Health purposes. This changes Temperature
reporting back to the default and instead configures Zone Status reporting
for every 5 minutes. The benefit of this is if the hub is offline when
the device sends a Zone Status Change Notification indicating moisture,
contact or motion, the attribute reporting will get the platform back in
sync with the current state within 5 minutes of the hub coming back online.

https://smartthings.atlassian.net/browse/SSVD-2858

### CODE REVIEW

**Comment by tpmanley** [comment]
@workingmonk @jackchi @darksun 

This change still needs testing but I wanted to get a quick review from you guys now. If it looks good I'll make the same change to Moisture, Multi, Open/Close, and Temp/Humidity.

**Comment by darksun** [comment]
This looks good to me 💯  looking forward to testing these changes.

**Comment by tpmanley** [comment]
@varzac

File: devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy [hunk 1]
```diff
@@ -244,11 +266,11 @@ private Map getTemperatureResult(value) {
...
 		value = v + offset
 	}
-    def descriptionText
-    if ( temperatureScale == 'C' )
-    	descriptionText = '{{ device.displayName }} was {{ value }}°C'
-    else
-    	descriptionText = '{{ device.displayName }} was {{ value }}°F'
+	def descriptionText
+	if ( temperatureScale == 'C' )
+		descriptionText = '{{ device.displayName }} was {{ value }}°C'
```
Comment (jackchi) [inline devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy:271]: Groovy string interpolation syntax is `${device.displayName}` or else this shows up as:
f3011b16-656c-4873-8efd-e71169aa1ddc  10:43:16 AM: debug Parse returned [name:water, value:dry, descriptionText:{{ device.displayName }} is dry, translatable:true]

f3011b16-656c-4873-8efd-e71169aa1ddc  10:43:16 AM: debug water
f3011b16-656c-4873-8efd-e71169aa1ddc  10:43:16 AM: debug description: zone status 0x0024 -- extended status 0x00

**Comment by jackchi** [comment]
👍  After log fixes.

File: devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy (hunk 1 above)
Comment (tpmanley) [reply devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy:271]: This is how internationalization was implemented. I just changed the spaces to tabs to be consistent with our coding style.

**Comment by varzac** [comment]
👍  looks good to me.

**Comment by workingmonk** [comment]
don't we need to do this for the other sensors too?

**Pull request closed without merging**